src/config/openapi.yml

```diff
@@ -183,7 +301,17 @@ paths:
           name: jolokia-session-id
           schema:
             type: string
-          required: true
+          required: false
+        - in: header
+          name: endpoint-name
+          schema:
+            type: string
+          required: false
+        - in: query
+          name: targetEndpoint
+          schema:
+            type: string
+          required: false
```

Comment:
I think this is a great addition to specify the actual endpoint the request is targeting. My question would be, why not having it directly a URL instead of a name?

Reply:
It could be changed to accepting both name and url. If it is a url passed in, use it to match the endpoint's url. If it's a name then it matches the name of the endpoint.

Reply:
Sounds good to me. We could improve the schema by specifying a regex to match either of those:

`^(https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&\/\/=]*)|\w+)$`

This one is matching a url using the https protocol or a word, we can drop the https protocol if needed, or have a list of supported protocols.

Reply:
good idea.